who_are_we.md

```diff
@@ -0,0 +1,111 @@
+
+For a year and a half, we worked along with Nir Yariv, the project manager, who later joined the Gizra team.
+
+It was our biggest client at the time, paying in dollars, and it gave us a lot of freedom to constantly iterate and polish our work. We were heavily invested for the first time in automatic testing. The tool we used was bare bone selenium. It was a horrible experience in terms of false positives and taming the automatic tests, but it showed us what we believed all along - real platforms can't be built without a certain amount of automatic test coverage.
+
+Even though it was our first international client, we were already work-life balance aware. The automatic tests made it possible to maintain it, as the platform hardly broke.
+
+The platform, originally starting in Spanish, didn't pick up, so Medico decided to translate it to English and Portuguese. They required an activity log that could be translated. That was the birth of the [Message ](https://www.drupal.org/project/message) module, later becoming a set of modules know as "the message stack".
+
+The site in Portuguese at a certain point picked up rapidly and we saw heavy use by Portuguese people. This sudden success in the Portuguese market was not enough to impress the investors who pulled the plug and didn't fund the second round.
+
+We were five people in the office when we were told that the money is going to end in a week or so.
+
+## Commerce Guys
+
+Luck, or asking the universe or whatever you decide to call it, brought Commerce Guys (CG) to us immediately after the work for Medico ended.
+
+We knew the owners for some time, and were always good friends that shared a similar technical vision. So that might have been the reason for them to ask Gizra to jump on board and help them get their [Commerce Kickstart](http://demo.commerceguys.com/) distribution up and running. The timeline was fierce given the amount of tasks they wanted to complete, and scrutinizing code reviews were in abundance. Just the way we liked it.
+
+If you look at our fancy Drupal's installation profile scripts you might not recognize it, but it all started with the work we did a _long_ time ago with CG.
+
+## Harvard University
+
+DrupalCon Denver 2012 arrived, and not long after Entity Reference was born (it started with Damien Tournoud from CG who later [asked Amitai](https://youtu.be/hC4RiK-m0BQ?t=7m40s) to become a co-maintainer) and was followed up with the complete rewrite of Organic groups for Drupal 7. You already read about the complete re-write for OG for Drupal 7? Yes you did. However the 7.1 version had flaws that we wanted to fix. So we went ahead and fixed all the architectural issues we wrongly committed in D7. This 7.2 is still up and running, and is the basis for OG for Drupal 8, that has similar concepts just different code (since Drupal 8 is so different from Drupal 7).
```

Comment:
Entity Reference was born a few month before Denver. It's a very important module, so I think people assume it was always there. But it wasn't.